release-19.2: acceptance: disable Elixir tests

otan opened the pull request:
Backport 1/1 commits from #52372.

/cc @cockroachdb/release

---

Elixir requires network to install dependencies. This can be changed
(painfully), but an upgrade an Elixir is causing even more problems. I'm
in favour of removing the acceptance (which I wrote) to fix this for now
as I have plentiful unit tests covering the same thing. Maybe someone
can come back to this if it's a large issue...

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/52373)
<!-- Reviewable:end -->

otan committed:
acceptance: disable Elixir tests

Elixir requires network to install dependencies. This can be changed
(painfully), but an upgrade on Elixir is causing even more problems. I'm
in favour of removing the acceptance (which I wrote) to fix this for now
as I have plentiful unit tests covering the same thing. Maybe someone
can come back to this if it's a large issue...

Release note: None
